cextern/cfitsio/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+PROJECT(CFITSIO)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.0)
+
+# Allow the developer to select if Dynamic or Static libraries are built
+OPTION (BUILD_SHARED_LIBS "Build Shared Libraries" ON)
+OPTION (USE_PTHREADS "Thread-safe build (using pthreads)" OFF)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}")
+set (LIB_DESTINATION "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}")
+set (INCLUDE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/include/")
+
+# Define project version
+SET(${PROJECT_NAME}_MAJOR_VERSION 3)
+SET(${PROJECT_NAME}_MINOR_VERSION 41)
+SET(${PROJECT_NAME}_VERSION ${${PROJECT_NAME}_MAJOR_VERSION}.${${PROJECT_NAME}_MINOR_VERSION})
+SET(LIB_NAME cfitsio)
+
+# Microsoft Visual Studio:
+IF(MSVC OR BORLAND)
+    # Define
+    ADD_DEFINITIONS(-D_CRT_SECURE_NO_DEPRECATE)
+    # Need an empty unistd.h in MSVC for flex-generated eval_l.c:
+    FILE(WRITE ${CMAKE_SOURCE_DIR}/unistd.h "")
```

Comment:
None of the `Makefile` get executed/included (certainly not now, I don't think before either - `setup_package` just includes `*.c` files), but obviously it should not be hard to mimic this in `setup_package`. I don't really understand, though, why this worked before...